.travis.yml

```
@@ -19,3 +21,19 @@ notifications:
 after_success:
   # push coverage results to Coveralls
   - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+
+jobs:
+  include:
+    # Make sure ChainRules.jl works with this
+    - state: Integration
+      name: ChainRules
+      julia: 1.0
+      os: linux
+      script:
+        - git clone "https://github.com/JuliaDiff/ChainRules.jl.git"
+        # Start julia with  the project set to newly cloned subdir ChainRules.jl
+        # dev the current directory to get the version of ChainRulesCore that we are testing
+        # update everything to resolve and sortout any versions,
+        # finally test the ChainRules.jl
+        - julia --project="ChainRules.jl" -e "using Pkg; Pkg.develop(\".\"); Pkg.update(); Pkg.test()"
```

```suggestion
       - julia --project="ChainRules.jl" -e "using Pkg; Pkg.pkg\"dev .\"; Pkg.pkg\"up\"; Pkg.pkg\"test\""
```

`Pkg.develop` requires a `PackageSpec` 
and this is just easier.